beacon-chain/rpc/prysm/v1alpha1/validator/proposer_epbs.go

```diff
@@ -31,6 +33,31 @@ func (vs *Server) SubmitSignedExecutionPayloadEnvelope(ctx context.Context, env
 	return nil, nil
 }
 
+// SubmitSignedExecutionPayloadHeader submits a signed execution payload header to the beacon node.
+func (vs *Server) SubmitSignedExecutionPayloadHeader(ctx context.Context, h *enginev1.SignedExecutionPayloadHeader) (*emptypb.Empty, error) {
+	if vs.TimeFetcher.CurrentSlot() != h.Message.Slot && vs.TimeFetcher.CurrentSlot() != h.Message.Slot-1 {
+		return nil, status.Errorf(codes.InvalidArgument, "invalid slot: current slot %d, got %d", vs.TimeFetcher.CurrentSlot(), h.Message.Slot)
```

Comment:
There are three calls for `vs.TimeFetcher.CurrentSlot()` in these two lines, can we change this to
```suggestion
        currSlot := vs.TimeFetcher.CurrentSlot()
	if currSlot > h.message.Slot || currSlot + 1  < h.message.Slot {
		return nil, status.Errorf(codes.InvalidArgument, "invalid slot: current slot %d, got %d", currSlot, h.Message.Slot)
```